File: scratch/find_views.js
const html = require('fs').readFileSync('public/command.html', 'utf8');
const matches = [...html.matchAll(/id=["']([^"']+)View["']/g)];
console.log("View IDs found in command.html:");
matches.forEach(m => console.log("- " + m[1] + "View"));

File: scratch/find_views.py
import re
with open('public/command.html', 'r', encoding='utf-8') as f:
    content = f.read()

matches = re.finditer(r'<[^>]*class="[^"]*ehr-view[^"]*"[^>]*>', content)
for m in matches:
    print(m.group(0))
